pkg/kubeapiserver/authenticator/config.go

```diff
@@ -174,13 +174,6 @@ func (config Config) New() (authenticator.Request, *spec.SecurityDefinitions, er
 		}
 
 		tokenAuthenticators = append(tokenAuthenticators, webhookTokenAuth)
-	} else {
-		// openshift gets injected here as a separate token authenticator because we also add a special group to our oauth authorized
-		// tokens that allows us to recognize human users differently than machine users.  The openAPI is always correct because we always
-		// configuration service account tokens, so we just have to create and add another authenticator.
-		// TODO make this a webhook authenticator and remove this patch.
-		// TODO - remove in 4.7, kept here not to disrupt authentication during 4.5->4.6 upgrade
-		tokenAuthenticators = AddOAuthServerAuthenticatorIfNeeded(tokenAuthenticators, config.APIAudiences)
```

Comment:
the diff doesn't show everything, but is the method `AddOAuthServerAuthenticatorIfNeeded` also removed?

Reply:
ah yes it is :+1:

Reply:
yes, in `pkg/kubeapiserver/authenticator/patch_authenticator.go`